### PR TITLE
Add meta parameters (Issue: MUR-10319)

### PR DESCRIPTION
**Pull request by lenaw**

### CODE REVIEW

File: services/config_service.lua [hunk 1]
```diff
@@ -26,6 +26,15 @@ if service.service == "sigfox" and (service.action == "added" or service.action
       if v.payloadConfig ~= nil then
         join(payloadConfigs, v.payloadConfig)
       end
+      if v.metadataConfig ~= nil then
+        for k, v in pairs(v.metadataConfig) do
+          if (k == "operatorName") then
+            join(payloadConfigs, {resource = k, definition = "char"})
```
Comment (sauvainr) [inline services/config_service.lua]: I think the join 2nd argument need one more wrapper.

Comment (lenaw) [reply services/config_service.lua]: yes, it should be an array. fixed.

**Comment by sauvainr** [comment]
@lenaw Can you check the failed test? If all good we can merge it.

**Comment by lenaw** [comment]
> @lenaw Can you check the failed test? If all good we can merge it.

seems like the failing is caused by qa python2 -> python3 problem, now it is fixed.

**Pull request merged**